experimental/PlaneCurve/ProjEllipticCurve.jl

```diff
@@ -0,0 +1,459 @@
+export ProjEllipticCurve, discriminant, issmooth, j_invariant,
+       Point_EllCurve, curve, weierstrass_form, toweierstrass,
+       iselliptic
+
+################################################################################
+# Helping functions
+################################################################################
+
+function isweierstrassform(eq::Oscar.MPolyElem{S}) where S <: FieldElem
```

Comment:
```suggestion
function isweierstrass_form(eq::Oscar.MPolyElem{S}) where S <: FieldElem
```
I know it is not exported but it is best to be consistent.